src/ucs/stats/stats.c

```diff
@@ -126,6 +184,7 @@ static void ucs_stats_node_remove(ucs_stats_node_t *node, int make_inactive)
     ucs_list_del(&node->list);
     if (make_inactive) {
         ucs_list_add_tail(&node->parent->children[UCS_STATS_INACTIVE_CHILDREN], &node->list);
+        node->cls = ucs_stats_get_class(node->cls);
```

Comment:
for example here, if ucs_stats_get_class() returned NULL (allocation failed), we can just fallback to ucs_stats_clean_node()

Reply:
node will be cleaned & destroyed